oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsStreamProcessor.java

```diff
@@ -199,4 +203,20 @@ private MetricsPersistentWorker downSamplingWorker(ModuleDefineHolder moduleDefi
 
         return persistentWorker;
     }
+
+    // todo javadoc
+    public void remove(ModuleManager manager, String metricName, Class<? extends Metrics> metricsClass) {
```

Comment:
Tip, we need to check whether the worker would be GC successfully to avoid potential OOM risk.

Reply:
Question, once this gets removed, we don't touch the storage, right?

Reply:
Yes, it is.